content/moderation/moderation-overview.md

```diff
@@ -0,0 +1,324 @@
++++
+archetype = "default"
+title = "Moderation"
+weight = 1
++++
+
+Everything in moderation, including moderation.
+
+<!--more-->
+
+![Overview of the moderation page](overview_moderation.png)
+
+<center>
+
+**1** Mod Log Channel **2** Report Channel **3** Moderation DM Error Channel **4** Clean Command **5** Report
+Command **6** Unban Logging **7** Giverole/Takerole commands **8** Give/Takerole Permitted Roles **9**
+Give/Takerole Logging **10** External Ban Logging **11** External Kick Logging **12** External Timeout Logging **13**
+Moderation Tabs **14** Bot Permissions Notice
+
+</center>
+
+## Moderation Page
+
+The moderation page allows configuration of common settings.
+
+### Mod Channels
+
+1. **Mod Log** - The mod log is the channel YAGPDB will send log messages to when most moderation commands are used. It
+includes username and user ID of both the moderator, and the moderated user. It additionally includes the reason for
+relevant commands (defaults to `<no reason provided>` when not provided), and a [log](/moderation/logging) of the
+channel where the command was run.
+2. **Report** - The channel for YAGPDB to send report messages to when users use the report command. Similar to the mod
+log, it includes username and user ID of the user reporting, and the user reported, as well as the reason and message
+log of the channel.
+3. **Moderation DM Error** - The channel where YAGPDB will send error messages from executing template script within
+[moderation DMs](#moderation-dms).
+
+### Command Toggles
+
+Certain potentially destructive moderation commands are disabled by default. These commands are:
+
+- The **clean command** (**4**) toggles the [clean command](/commands/all-commands#clean).
+- The **report command** (**5**) toggles the [report command](/commands/all-commands#report).
+- **Giverole/Takerole commands** (**7**) toggle the [giverole](/commands/all-commands#giverole) and
+[takerole](/commands/all-commands#takerole) commands.
+
+### Unban/Unmute Logging
+
+When the `unban`/`unmute` commands are used, YAGPDB will log their usage to the [mod log](#mod-channels) similarly to
+the other moderation commands, except without the logs.
```

Comment:
I am confused by the phrase "except without the logs" here.

Reply:
agreed, can omit